utils/contentful-posts.js

```diff
@@ -0,0 +1,17 @@
+import { createClient } from 'contentful';
+
+const spaceId = process.env.CONTENTFUL_SPACE_ID;
+const accessToken = process.env.CONTENTFUL_ACCESS_KEY;
+
+const client = createClient({
+  space: spaceId || '',
+  accessToken: accessToken || '',
+});
+
+async function fetchEntries() {
+  const entries = await client.getEntries({ content_type: 'blogPost' });
+  if (entries.items) return entries.items;
+  console.error(`Error getting Entries for ${contentType.name}.`);
```

Comment:
Is it enough to just `console.error` ? Maybe there's a better course of action here.

Let's think about the behavior of the app.
a) If I want the blog to render empty posts, maybe returning an empty array at the end if an option here.
Then, the React code can check if the posts is empty. In such scenario, we might show an appropriate message. E.g.
"There are no posts available right now".

b) Rendering a screen with empty posts may be a bad user experience or an error from the team managing Contentful's data. In this scenario, maybe throwing an exception is appropriate so that we fail the app completely and that way we warn our dev team to fix the error before publishing this state of the application.